core/src/main/resources/application.yml

```diff
@@ -1,6 +1,33 @@
 micronaut:
     application:
         name: core
+    security:
+        enabled: true
+        oauth2:
+            enabled: true
+            clients:
+                keycloak: 
+                    client-id: 'keycloak-client'
+                    openid:
+                        issuer: http://localhost:8090/auth/realms/Keycloak
```

Comment:
This should be configurable and set properly in `core-deployment.yaml`. Otherwise, deployment of our application in Google Cloud will stop working.